recipes/leftbar/style/leftbar_row.less

```diff
@@ -219,7 +219,7 @@
   }
 
   &__unread-badge {
-    display: var(--leftbar-row-omega-display);
+    display: var(--leftbar-row-omega-display) !important;
```

Comment:
What exactly is overriding this that we have to add important?

Reply:
![image](https://github.com/dialpad/dialtone-vue/assets/87546543/8a07e028-9606-4675-af80-6e9541054d4f)

![image](https://github.com/dialpad/dialtone-vue/assets/87546543/e728981c-f363-4d43-95c9-e1e073a9e260)

![image](https://github.com/dialpad/dialtone-vue/assets/87546543/78809e5e-319c-4af2-a0d1-3b6cabe268e1)

Styles from `.d-badge` coming from `dialtone.css` and `badge.less`.

Does `display: inline-flex` have priority over `display: var(--some-var)`?

Reply:
https://github.com/dialpad/dialtone-vue/assets/87546543/ceb944a8-d025-4da8-a334-b03ad61fb9c3

It doesn't make a lot of sense, seems that `.dt-leftbar-row__unread-badge` is actually being called after those other classes and they haven't more specificity, so the last display property should be applied, right?

Reply:
Hmm I wonder if it's the order it's imported since one is coming from dialtone and the other from dialtone vue. Maybe just hack more specificity into it rather than use important. Something like.  .d-badge.dt-leftbar-row...

Reply:
Since is CSS Im pretty sure the import order will affect this.
But why we have dialtone-vue styles in this?, This should be in dialtone and imported from there like all the components, Im missing sth here...

Im agree on try to find a proper fix for this, if we want to go with this hotfix we will probably miss a bunch of other issues we didn't find yet.

If we want to go with this kind of hotfixes we should document this approach somewhere...

Reply:
> But why we have dialtone-vue styles in this?

We don't put recipe components in dialtone, only core components

Reply:
hmm, okay so... recipe styles are overridden by core component styles.

Then we can confirm using !import in specific styles will not fix this issue, because this is a 'how we work styles' issue.

So Im guessing if we import dial tone-vue styles after dial tone styles these would be overwritten.

But could this bring us more issues?

Reply:
Ok I tested it locally in vue-3 branch and yes, the library override the component style

<img width="401" alt="image" src="https://github.com/dialpad/dialtone-vue/assets/89984179/765b74ae-a2ef-45d3-a000-588d26c89892">

Reply:
What did you change here? the import order of the files?

Reply:
I deleted the comment cos after realising what I did it was dumb, I just used the content-class prop of the component to fix it, but that is how it supposed to work.

Im doing test now with the current bug